helm-charts/splunk-otel-collector/values.yaml

```
@@ -400,6 +400,11 @@ agent:
     # so that agent pods can be discovered via dns etc
     enabled: false
 
+  # hostNetwork is an option that allows to use the networking stack of the host
+  # machine. Disabling this value will have affect on monitoring some control
+  # plane components. For Windows operating system the hostNetwork is disabled.
```

nits for clarity:

```suggestion
  # hostNetwork schedules the pod with the host's network namespace.
  # Disabling this value will affect monitoring some control plane
  # components. Enabling the agent service is recommended (see above).
  # Disregarded for Windows (unsupported by k8s).
```

It would be helpful to know the range of impacts so that they can be worked around.

---

that sounds clearer, updated 😊